general-concepts/licenses/text.xml

```diff
@@ -82,11 +82,38 @@ versions are specified elsewhere.
 </p>
 
 <p>
-Please watch for license conflicts. If the license indicated
-by the package is incompatible with the licenses used by its sources
-(e.g. BSD/MIT package including GPL sources), please contact
-the licenses team for guidance. Do not add packages that seem
-to include license term violations.
+If an installed file is built from sources using different licenses, all those
+licenses must be compatible. The effective license for the file will be the most
+restrictive of input licenses. For example:
+</p>
+
+<ol>
+	<li>
+		If a program includes both GPL- and BSD-licensed code, the license
+		for the resulting program will be respective GPL license since it is more
```

Comment:
"will be respective" sounds strange.

Reply:
Ok, I will reword the whole list to sound a little bit less strange.